Jamulus.pro

```diff
@@ -1,11 +1,21 @@
-VERSION = 3.6.2git
+VERSION = 3.6.2
```

Comment:
I assume this should be reverted/changed to *dev, but I'm not sure

Reply:
Definitely, will be added

Reply:
solved by #883 anyway